analyzer/src/funTest/assets/projects/external/directories-expected-output.yml

```diff
@@ -31,6 +31,8 @@ analyzer:
       - "Mozilla Public License 2.0"
       declared_licenses_processed:
```

Comment:
commit: Am I right that until now one design goal of the analyzer result (and scanner result as well) was `zero redundancy`? IIUC the data added by this change is redundant. Have you considered that?